pyproject.toml

```diff
@@ -10,8 +10,9 @@ authors = ["KyuWoo Choi <kyuwoo.choi@gmail.com>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
+python = ">=3.9,<3.12"
 mlflow = "^2.17.0"
+bentoml = "^1.3.10"
```

Comment:
Is there a reason for specifying a range instead of a specific version?

Reply:
The SDK needs to support diverse environments, each with its own dependencies. Fixing it to a single version could cause issues in environments where that version is unavailable. It's necessary to specify the widest possible range of versions in which this SDK can operate. In the future, as long as functionality is maintained, I'd like to review and extend support to a broader version range.